dev-core

```diff
@@ -76,25 +76,25 @@ if [[ ! $BINCACHED ]] || [[ ! $DOCKERCACHED ]]; then
   docker build -f Dockerfile.dev -t "mesg/engine:$VERSION" .
 fi
 
-# docker service rm core
+# docker service rm engine
 
 trap onexit EXIT INT
 
 function onexit {
   deleteServiceServer
-  docker service rm core
+  docker service rm engine
 }
 
-networkExists=$(docker network list -f name=^core$ -f driver=overlay -q)
+networkExists=$(docker network list -f name=^engine$ -f driver=overlay -q)
```

Comment:
This and the next 3 lines (creation of the engine network) should be moved before line 38 (creation of the system service server)

Reply:
Updated myself